flixel/util/FlxTimer.hx

```diff
@@ -273,4 +273,14 @@ class FlxTimerManager extends FlxBasic
 	{
 		FlxArrayUtil.clearArray(_timers);
 	}
-}
\ No newline at end of file
+	
+	/**
+	 * Calls the onComplete function of all timers, then removes them from the timer manager.
+	 */
+	public function completeAll():Void
+	{
+		for (timer in _timers)
+			timer.onComplete(timer);
```

Comment:
You can't iterate on timers while also removing them from the timer array (which `update()`->`cancel()` does), if this is the issue here. It's currently an issue in github, but you would have to manually `onComplete` all the timers and then `cancel` all the timers in another loop.

It's like when you kill an enemy; you loop through all the enemies, push the dead ones into a temp array, and then loop through the temp array and remove those from the main one.